bases/br_sp_alesp/despesas_gabinete/table_config.yaml

```diff
@@ -96,47 +114,51 @@ partitions: # REQUIRED
-
-    - name: cnpj
-      description: CNPJ ou CPF do favorecido
-      is_in_staging: True # Bool [True, False], whether the column is in the staging table
-      is_partition: False # Bool [True, False], whether the column is a partition.
-
-    - name: fornecedor
-      description: Nome do fornecedor/favorecido
-      is_in_staging: True # Bool [True, False], whether the column is in the staging table
-      is_partition: False # Bool [True, False], whether the column is a partition.
-
-    - name: tipo
-      description: Descrição do tipo da despesa
-      is_in_staging: True # Bool [True, False], whether the column is in the staging table
-      is_partition: False # Bool [True, False], whether the column is a partition.
-
-    - name: valor
-      description: Valor da despesa
-      is_in_staging: True # Bool [True, False], whether the column is in the staging table
-      is_partition: False # Bool [True, False], whether the column is a partition.
-
-    - name: ano
-      description: Ano a que se refere a despesa do gabinete
-      is_in_staging: True # Bool [True, False], whether the column is in the staging table
-      is_partition: True # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: mes
+        description: Mês a que se refere a despesa do gabinete
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: matricula
+        description: Número da matrícula do Deputado
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: deputado
+        description: Nome do Deputado Estadual
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: cnpj
+        description: CNPJ ou CPF do favorecido
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: fornecedor
+        description: Nome do fornecedor/favorecido
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: tipo
+        description: Descrição do tipo da despesa
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: valor
+        description: Valor da despesa
```

Comment:
Botar sempre `(R$)` na descrição se for tudo em reais. Se for em outras moedas pode acrescentar uma coluna `moeda`.